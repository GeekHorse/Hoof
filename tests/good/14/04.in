new down two done
move in
out
new down one done
move in
save
