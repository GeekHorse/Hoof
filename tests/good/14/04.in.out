new right done
new in one done
new down two done
out
